README.md

```diff
@@ -193,6 +202,28 @@ The autograder failed to grade your assignment.
 Message from the autograder: Request could not be authenticated. Ensure that your username, password, and course are properly set.
 ```
 
+##### Submitting an Assignment Late
+
+If you are submitting an assignment late, the autograder requires confirmation in order to grade your submission.
+This helps users avoid situations where they accidentally submit an assignment late or submit to the wrong assignment.
+Users must add the `--allow-late` flag to the normal submission command when they want to submit an assignment past the due date.
+
+Here is a sample output when trying to submit a late assignment without the flag:
```

Comment:
Stay closer to the patter used in the rest of this doc for example output.
Something like "For example, your output when submitting a late assignment may look like:".
This is a little more abstract and gives us wiggle room if the output changes.